### PR TITLE
DOC-9431 Feedback: Overload Dashboard

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-9431

(1) On Overload Dashboard page, added/replaced graphs for 23.2 and added Goroutine Scheduling Latency: 99th percentile graph which existed in v23.1 and v22.2. (2) On v23.2.0-alpha.2 page, added release note for PR 110135 and fixed typo. (3) On Admission Control, Common Issues to Monitor, and Storage Layer pages, replaced link from LSM L0 Health graph to IO Overload graph. (4) Updated healthy-lsm include file to refer to IO Overload value.

Rendered pages:
- [v23.2 Release Notes](https://deploy-preview-18193--cockroachdb-docs.netlify.app/docs/releases/v23.2.html#v23-2-0-alpha-2-db-console-changes)
- [Overload Dashboard](https://deploy-preview-18193--cockroachdb-docs.netlify.app/docs/v23.2/ui-overload-dashboard.html#io-overload)

### CODE REVIEW

**Comment by florence-crl** [comment]
@aadityasondhi please review these docs changes. Additionally please advise how I should change this warning:
> **Expected values for a healthy cluster**: The number of L0 files should **not** be in the high thousands. High values indicate heavy write load that is causing accumulation of files in level 0. These files are not being compacted quickly enough to lower levels, resulting in a misshapen LSM.

Since **IO Overload** no longer uses the raw metric, how should I replace:
> The number of L0 files should not be in the high thousands.
?

File: src/current/v23.2/admission-control.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This is particularly important for CockroachDB {{ site.data.products.serverless
 Admission control can help if your cluster has degraded performance due to the following types of node overload scenarios:
 
 - The node has more than 32 runnable goroutines per CPU, visible in the **Runnable goroutines per CPU** graph in the [**Overload** dashboard]({% link {{ page.version.version }}/ui-overload-dashboard.md %}#runnable-goroutines-per-cpu).
-- The node has a high number of files in level 0 of the Pebble LSM tree, visible in the **LSM L0 Health** graph in the [**Overload** dashboard]({% link {{ page.version.version }}/ui-overload-dashboard.md %}#lsm-l0-health).
+- The node has a high number of files in level 0 of the Pebble LSM tree, visible in the **IO Overload** graph in the [**Overload** dashboard]({% link {{ page.version.version }}/ui-overload-dashboard.md %}#io-overload).
```
Comment (aadityasondhi) [inline src/current/v23.2/admission-control.md:37]: Maybe we can reword this to remove "high number of files in level 0" since that is not user visible anymore. 

We can instead say something along the lines of, "The node has a high amount of overload in the pebble LSM tree, visble in the **IO Overload** graph..."

**Review by aadityasondhi — APPROVED** [review]
LGTM with one minor comment.

To answer your question:

> Since IO Overload no longer uses the raw metric, how should I replace:

>>The number of L0 files should not be in the high thousands.
?

A score greater than 1.0 generally indicates an overload in the Pebble LSM tree.

File: src/current/v23.2/admission-control.md (hunk 1 above)
Comment (florence-crl) [reply src/current/v23.2/admission-control.md]: updated

File: src/current/_includes/v23.2/prod-deployment/healthy-lsm.md [hunk 2]
```diff
@@ -1 +1 @@
-**Expected values for a healthy cluster**: The number of L0 files should **not** be in the high thousands. High values indicate heavy write load that is causing accumulation of files in level 0. These files are not being compacted quickly enough to lower levels, resulting in a misshapen LSM.
\ No newline at end of file
+**Expected values for a healthy cluster**: An IO Overload value greater than 1.0 generally indicates an overload in the Pebble LSM tree. High values indicate heavy write load that is causing accumulation of files in level 0. These files are not being compacted quickly enough to lower levels, resulting in a misshapen LSM.
```
Comment (rmloveland) [inline src/current/_includes/v23.2/prod-deployment/healthy-lsm.md:1]: "misshapen LSM" could be a link to https://www.cockroachlabs.com/docs/v23.2/architecture/storage-layer#inverted-lsms

**Review by rmloveland — APPROVED** [review]
LGTM! had one link suggestion
